share/eds/testdata/example-root.json

```diff
@@ -1,22 +1,22 @@
 {
 "row_roots": [
-"A3I8MUKAXaEf7JDjUxllL5CGD0/uS8k2l3KpbN+u5fa3tGL1h9FSqZhCJFcZhzAT",
-"IA74YwcC1tFMkQfD9sE2NO7hwDtjo37O1KiiptYPCgAQWQnv/G7bxu6mUJ2s3Ljk",
-"ZskuRtu6YfK9pTqLAOsfLXbvjrvDD+80xoj/850Z4YGvmAaRH5vFXraZNWgEQNyr",
-"w9Tz/Q7t/JfX7McKTWvNjhqi784y+nGCYXrQtu4jy90kx0irPJcVTXTQ7v6Xnjdf",
-"/////////////////////1MCHZjf0ySoNlpt+h8Vb6QFm7BHYJXI14SMiNRgwA0K",
-"/////////////////////2dOgrZG2ZT+KIrfjoPmTcGpqXdsdFZE6ZHcF0l+fL6i",
-"/////////////////////7OJEeOYA6VsagglTnpIysAtyK3m3u4d8Zd66FuqX0+M",
-"/////////////////////+hYZyJ10gx7Brp05g/MLtSV4z7SLLwGcMsXZI/Fgw0J"
+"AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAHTFhSx3pL2TzSAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA471vB8w31XZHnopsd0qqnrIfVzBnaeUwwhrSzNvBV0VV+fR488sbA3k4=",
```

Comment:
the example had to be updated b/c row_roots and col_roots are now longer.

Before: 2 * 8 bytes (namespace) + 32 bytes = 48 bytes
After: 2 * 33 bytes (namespace) + 32 bytes = 98 bytes

Reply:
> AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAH